<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>7angram</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagenes.qrc">
    <normaloff>:/imagenes/Logos/icono.png</normaloff>:/imagenes/Logos/icono.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background-color: rgb(255, 255, 255);
}

#startYouAdventure{
color: #999;
}

#iconoPlay, #iconoInstructions, #iconoExit{
background-color: #FFF;
border: none;
}

#playButton{
background-color: #FFF;
color: #777;
border: none;
font-size: 100px;
font-weight: 600;
padding: 5px;
text-align: center;
}

#playButton:hover{
background-color: #FFF;
font-size: 120px; 
color: #0A0;
font-weight: 900
}

#instructionsButton{
background-color: #FFF;
color: #777;
border: none;
font-size: 60px;
font-weight: 600;
padding: 5px;
text-align: center;
}

#instructionsButton:hover{
background-color: #FFF;
font-size: 63px; 
color: #007;
font-weight: 900
}

#exitButton{
background-color: #FFF;
color: #777;
border: none;
font-size: 60px;
font-weight: 600;
padding: 5px;
text-align: center;
}

#exitButton:hover{
background-color: #FFF;
font-size: 80px; 
color: #F00;
font-weight: 900
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="startYouAdventure">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START YOUR ADVENTURE!</string>
    </property>
   </widget>
   <widget class="QLabel" name="logoTangram">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>941</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imagenes.qrc">:/imagenes/Logos/7ANGRAM LOGO A COLOR.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="iconoFlecha">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imagenes.qrc">:/imagenes/Iconos/FlechaTransparente.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="iconoTangram">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>331</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imagenes.qrc">:/imagenes/Iconos/TANGRAM.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="iconoPlay">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#iconoPlay{
image: url(:/imagenes/Iconos/Play.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="iconoInstructions">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#iconoInstructions{
	image: url(:/imagenes/Iconos/Instructions.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="iconoExit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>430</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#iconoExit{
	image: url(:/imagenes/Iconos/Salir.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="instructionsButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>INSTRUCTIONS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>430</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>200</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>V 8.0</string>
    </property>
   </widget>
   <zorder>playButton</zorder>
   <zorder>instructionsButton</zorder>
   <zorder>exitButton</zorder>
   <zorder>startYouAdventure</zorder>
   <zorder>logoTangram</zorder>
   <zorder>iconoFlecha</zorder>
   <zorder>iconoTangram</zorder>
   <zorder>iconoPlay</zorder>
   <zorder>iconoInstructions</zorder>
   <zorder>iconoExit</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
